src/gmenu2x.cpp

```diff
@@ -668,6 +669,10 @@ void GMenu2X::showSettings() {
 			*this, tr["Output logs"],
 			tr["Logs the output of the links. Use the Log Viewer to read them."],
 			&confInt["outputLogs"])));
+	sd.addSetting(unique_ptr<MenuSetting>(new MenuSettingBool(
+			*this, tr["Trim extensions"],
+			tr["Trims extensions in the file lister."],
```

Comment:
`Trims extensions in the file lister.` -> `Trim file extensions in the file explorer.`